engine/client/cl_parse.c

```diff
@@ -1696,7 +1699,30 @@ CL_ParseVoiceData
 */
 void CL_ParseVoiceData( sizebuf_t *msg )
 {
-	// TODO: ???
+	int size, idx, frames;
+	unsigned char received[8192];
+
+	idx = MSG_ReadByte( msg ) + 1;
+
+	frames = MSG_ReadByte( msg );
+
+	size = MSG_ReadShort( msg );
+	size = Q_min( size, 8192 );
```

Comment:
Use `sizeof( received )` here

Reply:
Maybe allocating such a big buffer on stack is a bad idea? I can make it static

Reply:
Not at all?

Reply:
I made it as static, keep as it is?

Reply:
Use sizeof. It's fine to have 8k buffer on stack, I think we have somewhere even more than that.

Reply:
Ok fixed